public/javascripts/vector-render.js

```diff
@@ -23,6 +23,9 @@ function vec2_angle(v) {
 function render_vector_drawing(a, padding) {
   var shape = a.shape || "";
   var path = [];
+  if(typeof a.control_points == 'string'){
+    a.control_points = eval(a.control_points);
```

Comment:
don't use `eval`, which is a security risk. use JSON.parse() instead.

Reply:
actually, please move this to another PR, it has nothing to do with puppeteer directly.